pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.ruff]
+# Ignore some well known paths
+exclude = [
+  ".tox",
+  ".venv",
+  "*.egg",
+  "build",
+  "db/env.py",
+  "db/versions/*.py",
+  "dist",
+  "doc",
+  "site"
+]
+line-length = 120
+select = ["E", "F", "I", "PLC", "PLE", "UP"]
+target-version = "py37"
```

Comment:
As GitHub format is not really something what would be annoying to use, I think we can stick with it being default here as well.

Reply:
Did you try running ruff on the command line with `--format=github` and some errors?